R/AllGenerics.R

```diff
@@ -411,14 +411,21 @@ setGeneric("chromPeakData<-", function(object, value)
 #'
 #' Parameter `return.type` allows to specify the *type* of the result object.
 #' With `return.type = "Spectra"` (the default) a [Spectra] object with all
-#' matching spectra is returned. The spectra variable `"peak_id"` of the
-#' returned `Spectra` contains the ID of the chromatographic peak (i.e., the
-#' rowname of the peak in the `chromPeaks` matrix) for each spectrum.
-#' With `return.type = "Spectra"` a `List` of `Spectra` is returned. The
-#' length of the list is equal to the number of rows of `chromPeaks`. Each
-#' element of the list contains thus a `Spectra` with all spectra for one
-#' chromatographic peak (or a `Spectra` of length 0 if no spectrum was found
-#' for the respective chromatographic peak).
+#' matching spectra is returned. With `return.type = "Spectra"` a `List` of
+#' `Spectra` is returned. The length of the list is equal to the number of rows
+#' of `chromPeaks`. Each element of the list contains thus a `Spectra` with all
+#' spectra for one chromatographic peak (or a `Spectra` of length 0 if no
+#' spectrum was found for the respective chromatographic peak).
+#'
+#' Parameters `addColumnsChromPeaks` allow the user to add specific metadata
```

Comment:
or should we call the parameter `chromPeakColumns`? Wouldn't `chromPeakSpectra(x, chromPeakColumns = c("rt", "rtmin"))` feel a bit more natural than `chromPeakSpectra(x, addColumnsChromPeaks = c("rt", "rtmin"))`?

but I don't have a strong opinion; happy to discuss.

Reply:
I'm going with chromPeakColumns() it makes complete sense in my opinion